modules/developers-guide/pages/graphql-using.adoc

```diff
@@ -0,0 +1,519 @@
+= Using the Stargate GraphQL API
+
+Stargate is a data gateway deployed between client applications and a database.
+The GraphQL API plugin that exposes CRUD access to data stored in Cassandra tables.
+
+== Using the GraphQL Playground
+
+The easiest way to get started is to use the built-in GraphQL playground that is
+included in the `stargate` docker container. It can be accessed at
+http://localhost:8080/playground[http://localhost:8080/playground] using a local
+browser after the docker container is started.
+
+Once in the playground, you can create new schema and interact with the
+GraphQL APIs. The server paths are structured to provide access to creating and
+querying schema, as well as querying and modifying Cassandra data:
+
+* `/graphql-schema`
+** An API for exploring and creating schema, or Data Definition Language (DDL).
+For example, Cassandra has queries to create, modify, drop keyspaces and tables,
+such as `CREATE KEYSPACE`, `CREATE TABLE1`, or `DROP TABLE`.
+* `/graphql/<keyspace>`
+** An API for querying and modifying your Cassandra tables using GraphQL fields.
+
+We'll start the playground with `/graphql-schema` to create some schema.
+
+== Creating schema
+
+=== Creating a keyspace using the GraphQL API
+
+Before you can start using the GraphQL API, you must first create a Cassandra
+keyspace and at least one table in your database. If you are connecting to a
+Cassandra database with existing schema, you can skip this step.
+
+Inside the GraphQL playground, navigate to
+http://localhost:8080/graphql-schema[http://localhost:8080/graphql-schema]
+and create a keyspace by executing the following mutation:
+
+[source, plaintext]
+----
+include::quickstart:partial$graphql/createKeyspace.graphql[]
+----
+For each keyspace created in your Cassandra schema, a new path is created under
+the `graphql-path` root (default is: `/graphql`). For example, the mutation just
+executed creates a path `/graphql/library` for the `library` keyspace when
+Cassandra creates the keyspace.
+
+Add the auth token to the HTTP Headers box in the lower lefthand corner:
+
+[source, plaintext]
+----
+{
+  "x-cassandra-token":"bff43799-4682-4375-99e8-23c8a9d0f304"
+}
+----
+
+IMPORTANT: Notice that the key for this JSON token is different than the value that the
+generate token has. It is `x-cassandra-token`, not `auth-token`.
+
+You should see a return value of:
+
+[source, plaintext]
+----
+NEED THE VALUE AFTER createKeyspace is VALID
```

Comment:
need to update this

Reply:
oops. I've fixed it.

Reply:
That's another reason I need to put all code in partials/examples and just cross-reference.